zerver/tests/test_migrations.py

```diff
@@ -31,7 +31,7 @@
 
 
 class MessageEditHistoryLegacyFormats(MigrationsTestCase):
-    __unittest_skip__ = False
+    __unittest_skip__ = True
```

Comment:
`__unittest_skip__` is not a public interface. That’s what the [`unittest.skip`](https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures) decorator is for.

We have some existing code using `__unittest_skip__`. I suspect it’s to work around https://bugs.python.org/issue17519. We should find a better workaround there. But in any case, _this_ use of `__unittest_skip__` doesn’t appear to be related to that, so should be replaced with the decorator.